amqp-utils/pom.xml

```diff
@@ -8,6 +8,10 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>amqp-utils</artifactId>
+  <properties>
+    <maven.compiler.source>8</maven.compiler.source>
```

Comment:
I think source can be left at 11?

Reply:
Probably yes.. it is not something I have ever needed to do, but it should work.